_posts/2018-07-25-Relay-Networking-Deep-Dive.markdown

```diff
@@ -291,31 +346,47 @@ const batchRequestQuery = async (
 };
 ```
 
-`getDeferrableVariables` function will get variables from result data from other requests.
+## Relay Modern is very flexible
 
-`batchRequestQuery` function will execute each of the requests, and will `sink.next()` as soon as it has the GraphQL
-server response data. It will only close the Observable stream when all requests has been fullfiled.
+Depending on your application needs, you can scale from a simpler Promise-based API for your custom network layer to
+one that uses Observables to always resolves from cache data first and then resolves from the server.
 
---
+Here are some production examples:
 
-Relay Modern is very flexible!
+- [Artsy Emission][artsy]: Uses the Promise API, caches the results locally, and shares logic with native code in an
+  iOS app so that queries can be pre-cached before the JavaScript runtime has started.
 
-You can have a custom network layer that uses observables to always resolves from cache data first and then resolves
-from the server.
+- [ReactRelayNetworkModern][]: A network layer that uses the middleware pattern to separate responsibilities like
+  retrying, logging, caching and auth.
 
-You can implement offline first apps using a custom network layer.
+- [timobetina's example][timobetina]: The simplest Observable network layer you can start with.
+
+<!-- TODO: More, @sibelius do you have some good examples? -->
 
 ## More Resources
 
-If you don't know GraphQL or you want to improve it, take a look on our boilerplate that uses dataloader to batch
-and cache requests to database https://github.com/entria/graphql-dataloader-boilerplate
+If you want to expand your understanding of GraphQL and Relay Modern, I have two great related resources:
+
+- A boilerplate that uses dataloader to batch and cache requests to your database in a GraphQL API:
+  https://github.com/entria/graphql-dataloader-boilerplate
+
+- A simple boilerplate for working with Relay Modern and React Navigation:
+  https://github.com/entria/ReactNavigationRelayModern
```

Comment:
Switched it out to a list because DataLoaders on the API and a Relay Modern + React Nav on the client aren't easily connected